buildbot/MANIFEST.cfg

```diff
@@ -63,7 +63,7 @@ nose.tests = anybox.recipe.openerp/tests_with_openerp --logging-level=INFO
 watch = git https://github.com/anybox/anybox.recipe.odoo a.r.openerp-1.9
 
 [recipe-odoo-80]
-# A buildout to test latest version of the OpenERP recipe for odoo 8.0
+# A buildout to test latest version of the Odoo recipe for odoo 8.0
 buildout = git https://github.com/anybox/anybox.recipe.odoo a.r.openerp-1.9 recipe-odoo-80.cfg subdir=buildbot
```

Comment:
No, this one was correct: a.r.openerp-1.9 is anybox.recipe.openerp (it's already hard enough to be clear), please revert it.